src/pybind/mgr/dashboard/frontend/applitools.config.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  appName: 'Ceph Dashboard',
+  apiKey: process.env.APPLITOOLS_API_KEY,
+  browser: [
+    { width: 1920, height: 1080, name: 'chrome' },
+    { width: 1920, height: 1080, name: 'firefox' },
+    { width: 800, height: 600, name: 'chrome' },
```

Comment:
@aaryanporwal @epuertat Do we really want to support 800x600 screen resolution?

Reply:
Yeah I think it's better to use a resolution with width less than 768 px, because that's when the left navigation bar collapses.